en/components/label.md

```diff
@@ -1,45 +1,52 @@
-|Font Family| Font family name, takes effect when using system font.
-|Cache Mode| The text cache mode (New in v2.0.9). Takes effect only for **system font** or **ttf** font, BMFont do not require this optimization. And includes **NONE**, **BITMAP**, **CHAR** three modes. See [Cache Mode](#cache-mode-new-in-v209) below for details.
-|  Use System Font | Boolean value, choose whether to use the system font or not.
+| Properties       | Function Explanation |
+| ---------------- | -------------------- |
+| String           | Text content character string.                                                                                              |
+| Horizontal Align | Horizontal alignment pattern of the text. The options are LEFT, CENTER and RIGHT.                                           |
+| Vertical Align   | Vertical alignment pattern of the text. The options are TOP, CENTER and BOTTOM.                                             |
+| Font Size        | Font size of the text.                                                                                                      |
+| Line Height      | Line height of the text.                                                                                                    |
+| SpacingX         | The spacing between font characters, only available in BMFont.                                                              |
+| Overflow         | Layout pattern of the text. Currently supports CLAMP, SHRINK and RESIZE_HEIGHT. See [Label Layout](#label-layout) below for more detailed information. |
+| Enable Wrap Text | Enable or disable the text line feed. (which takes effect when the Overflow is set to CLAMP or SHRINK)                      |
+| Font             | Specify the font file needed for text rendering. If the **System Font** is used, then this attribute can be set to `null`.  |
+| Font Family| Font family name, takes effect when using **System Font**.                                                                        |
+| Enable Bold      | If enabled, the bold effect is added to the text. Takes effect when using **System Font** or **TTF** font.                  |
+| Enable Italic    | If enabled, the italic effect is added to the text. Takes effect when using **System Font** or **TTF** font.                |
+| Enable Underline | If enabled, the underline effect is added to the text. Takes effect when using **System Font** or **TTF** font.             |
+| Underline Height | Height of underline.                                                                                                        |
+| Cache Mode       | The text cache mode. Takes effect only for **System Font** or **TTF** font, BMFont do not require this optimization. And includes **NONE**, **BITMAP**, **CHAR** three modes. See [Cache Mode](#cache-mode-new-in-v209) below for details.                                                   |
```

Comment:
do -> does

Reply:
The text cache mode.  -> Text cache mode includes **NONE**, **BITMAP** or **CHAR**.